code/datums/components/magnetic_catch.dm

```diff
@@ -0,0 +1,23 @@
+/datum/component/magnetic_catch/Initialize()
```

Comment:
a magic datum that makes things not throwable?

this is silly just make an ingame way to secure things lol what is this even for

Reply:
I suggested this, since it also means we can have a player-available item that applies the component in the future.

Also because components are nice.

Reply:
@fludd12 this was a part of the pr from the beginning kek

It stops anything on the tile from being thrown, not stop the thing it's applied to from being thrown @optimumtact

Reply:
oh
ok
i didn't realize

/////

Reply:
tables and racks to prevent the stuff on them from being thrown off

Reply:
okay proceed